modules/Sources/Features/SendFlow/SendFlowStore.swift

```diff
@@ -79,7 +79,8 @@ public struct SendFlowReducer: Reducer {
         public var isValidForm: Bool {
             @Dependency(\.zcashSDKEnvironment) var zcashSDKEnvironment
 
-            return transactionAmountInputState.amount.data > zcashSDKEnvironment.network.constants.defaultFee().amount
```

Comment:
This validity check was confirming that the user was not sending a transaction worth less than the fee. Was this removed to enable users to send zero-value shielded transactions (i.e. paying a fee to send just a memo to a recipient)? I don't actually know what receiving wallets will do in this case, because zero-value shielded outputs are dummy notes, and I would not be surprised if they get filtered out somewhere.

In any case, it's unclear to me why this change was made in a PR about the memo field behaviour for transparent addresses.

Reply:
I actually reverted the code because any change here is put of the scope of this ticket. If any change is needed we will create a follow up ticket.